README.md

```diff
@@ -255,6 +257,30 @@ fastify.register(require('fastify-swagger'), {
 }
 ```
 
+##### Asynchronous transform
```

Comment:
Can you mention about if you use `async transform`, you need to await `fastify.swagger`. Otherwise, it will return `Promise` and may break your application.